taxi/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    class Meta:
+        ordering = ["username"]
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name} ({self.username})"
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name} ({self.country})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.SET_NULL,

```

Comment:
it's better to use `models.CASCADE` to delete cars when the manufacturer is deleted

Reply:
I think it is bad idea.
We should not lose machines if the manufacturer is removed. Also the manufacturer may not be known.
But if your disagree, i'll change...

Reply:
CASCADE always worries me, can I use PROTECTED?

Reply:
let it be

Reply:
but why CASCADE always worries you? It's a common on_delete option

Reply:
Because it can cause data loss